Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -155,9 +152,8 @@ void JitArm64::reg_imm(u32 d, u32 a, u32 value, u32 (*do_op)(u32, u32),
   else
   {
     gpr.BindToRegister(d, d == a);
-    ARM64Reg WA = gpr.GetReg();
+    auto WA = gpr.GetScopedReg();
     (this->*op)(gpr.R(d), gpr.R(a), value, WA);
-    gpr.Unlock(WA);
```

Comment:
ComputeRC0 allocates a register, so it's beneficial to unlock this register before calling ComputeRC0.